mkosi/distribution/opensuse.py

```diff
@@ -213,30 +214,22 @@ def repositories(cls, context: Context) -> Iterable[RpmRepository]:
                         enabled=False,
                     )
 
-            if context.config.release in ("current", "stable", "leap"):
-                url = join_mirror(mirror, f"{subdir}/update/openSUSE-current")
-                yield RpmRepository(
-                    id="oss-update",
-                    url=f"baseurl={url}",
-                    gpgurls=fetch_gpgurls(context, url) if not zypper else (),
-                )
+            if (
+                context.config.release in ("current", "stable", "leap")
+                or GenericVersion(context.config.release) >= 16
```

Comment:
Relying on the fact that tumbleweed compares lower than 16 here seems very opaque. Please double check that the release is a digit before doing version comparisons on it.